packages/expo-image/android/src/main/java/expo/modules/image/drawing/BorderDrawable.kt

```diff
@@ -634,64 +635,68 @@ class BorderDrawable(private val mContext: Context) : Drawable() {
     /** Compute mInnerTopLeftCorner  */
     mInnerTopLeftCorner!!.x = mInnerClipTempRectForBorderRadius!!.left
     mInnerTopLeftCorner!!.y = mInnerClipTempRectForBorderRadius!!.top
-    getEllipseIntersectionWithLine( // Ellipse Bounds
-      mInnerClipTempRectForBorderRadius!!.left.toDouble(),
-      mInnerClipTempRectForBorderRadius!!.top.toDouble(), (
-      mInnerClipTempRectForBorderRadius!!.left + 2 * innerTopLeftRadiusX).toDouble(), (
-      mInnerClipTempRectForBorderRadius!!.top + 2 * innerTopLeftRadiusY).toDouble(),  // Line Start
-      mOuterClipTempRectForBorderRadius!!.left.toDouble(),
-      mOuterClipTempRectForBorderRadius!!.top.toDouble(),  // Line End
-      mInnerClipTempRectForBorderRadius!!.left.toDouble(),
-      mInnerClipTempRectForBorderRadius!!.top.toDouble(),  // Result
-      mInnerTopLeftCorner!!)
+    getEllipseIntersectionWithLine(
+        // Ellipse Bounds
+        mInnerClipTempRectForBorderRadius!!.left.toDouble(),
+        mInnerClipTempRectForBorderRadius!!.top.toDouble(), (
+        mInnerClipTempRectForBorderRadius!!.left + 2 * innerTopLeftRadiusX).toDouble(), (
+        mInnerClipTempRectForBorderRadius!!.top + 2 * innerTopLeftRadiusY).toDouble(), Line Start
```

Comment:
```suggestion
        mInnerClipTempRectForBorderRadius!!.top + 2 * innerTopLeftRadiusY).toDouble(), // Line Start
```
The project doesn't compile right now 😜